_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -11962,6 +11961,9 @@
 /obj/effect/turf_decal/tile/blue{
 	dir = 8
 	},
+/obj/structure/closet/crate/secure/clown{
+	name = "Toybox 4.0"
```

Comment:
All of your map changes has them manually renamed to Toybox 4.0.
Create a new typepath that just uses that as a name instead